airflow/settings.py

```diff
@@ -576,6 +576,13 @@ def initialize():
     atexit.register(dispose_orm)
 
 
+def is_telemetry_collection_enabled() -> bool:
+    """Check if scarf analytics is enabled."""
+    return conf.getboolean("telemetry_collection", "enabled", fallback=True) and (
+        os.getenv("SCARF_ANALYTICS") != "false"
```

Comment:
```suggestion
        os.getenv("SCARF_ANALYTICS").strip().lower() != "false"
```
could be helpful in case someone sets it to `False` or `FALSE` etc; and/or adds some leading/trailing whitespaces.

Reply:
Good point

Reply:
fixed in https://github.com/apache/airflow/pull/39510/commits/93c94e6b48a93787654ff4c712681ed3945060fe